cscs-checks/tools/profiling_and_debugging/notool.py

```diff
@@ -6,14 +6,14 @@
 
 
 @rfm.required_version('>=2.14')
-@rfm.parameterized_test(['C'], ['C++'], ['F90'])
+@rfm.parameterized_test(['C++'], ['F90'])
 class JacobiNoToolHybrid(rfm.RegressionTest):
     def __init__(self, lang):
         super().__init__()
-        self.name = 'jacobi_%s' % lang.replace('+', 'p')
-        self.descr = '%s check' % lang
+        self.name = '%s_%s' % (type(self).__name__, lang.replace('+', 'p'))
+        self.descr = 'Jacobi (without tool) %s check' % lang
         self.language = lang
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc']
+        self.valid_systems = ['dom:gpu', 'dom:mc']
```

Comment:
You should keep `daint`, because otherwise you will have to change the test later. This version will not run on Daint.